manifests/h/HP/ImageAssistant/5.1.11/HP.ImageAssistant.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: HP.ImageAssistant
+PackageVersion: 5.1.11
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://hpia.hpcloud.hp.com/downloads/hpia/hp-hpia-5.1.11.exe
+  InstallerSha256: 504D44012A4139DE9A42864A1A19670012F2C57224AF777FB5A1CB2F4E561058
+  InstallModes:
+  - silent
+  InstallerSwitches:
+    Silent: /s
+    SilentWithProgress: /s
+    InstallLocation: /e /f <INSTALLPATH>
+  InstallerSuccessCodes:
+  - 1168

```

Comment:
When invoking `hp-hpia-5.1.11.exe /s /e /f <INSTALLPATH>` directly, no outputs from the installer and the exit code is 0. But when testing locally by `winget install -m <path> -l <INSTALLPATH>`, installer will raise a `0x8000ffff` error and exit with code 1168. All binary files are extracted to `<INSTALLPATH>` and can work as expected.

Add `InstallerSuccessCodes` here to fix the problem.